includes/wikia/helpers/FileNamespaceSanitizeHelper.php

```diff
@@ -37,6 +37,11 @@ private function getFilePrefixRegex( $contLang ) {
 					$fileNamespaces [] = $alias;
 				}
 			}
+
+			$fileNamespaces = array_map(function( $namespace ) {
+				return mb_ereg_replace('_', '(_|\ )', $namespace);
```

Comment:
Please add a comment here :)
(that we want to match user-provided filename that may contain both spaces and underscores)